src/components/Person/Person.jsx

```diff
@@ -1 +1,42 @@
-// export const Person = ({ person }) => ();
+const MALE = 'm';
+const WIFE = 'wife';
+const HUSBAND = 'husband';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const getPartner = () => {
+    if (isMarried) {
+      return sex === MALE
+        ? WIFE
+        : HUSBAND;
+    }
+
+    return 'I am not married';
```

Comment:
'I'm not married' doesn't sound like partner, I'd prefer to create variable and use ternary